content/models.py

```diff
@@ -38,6 +38,10 @@ class Meta:
 
 content_blocks = rich_text_blocks + [
     ('two_columns', TwoColumnBlock()),
+    ('collapsible', blocks.RichTextBlock(icon="collapse-down",
+                                        label = _('Collapsible paragraph'),
+                                        template = 'content/blocks/collapsible.html',
+                                        )),
```

Comment:
This should be formatted as follows:
```
     ('collapsible', blocks.RichTextBlock(
        icon="collapse-down",
        label=_('Collapsible paragraph'),
        template='content/blocks/collapsible.html',
    )),
```
I.e. change indentation and remove spaces around keyword arguments.

Reply:
This done